src/abstract-sql-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Definition } from '@resin/odata-to-abstract-sql';
+
+import * as _ from 'lodash';
+import { readFileSync } from 'fs';
+import { sbvrUtils } from '@resin/pinejs';
+import { AbstractSqlModel } from '@resin/abstract-sql-compiler';
+
+export const generateAbstractSql = (seModelPath: string): AbstractSqlModel => {
```

Comment:
```suggestion
export const generateAbstractSqlModel = (seModelPath: string): AbstractSqlModel => {
```

I have been confused before between an `AbstractSqlModel` and `AbstractSqlQuery` so it would be good to be explicit in the method names, I think.